unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -136,6 +137,7 @@ function cert_generateFilebeatcertificates() {
             eval "openssl req -new -nodes -newkey rsa:2048 -keyout /tmp/wazuh-certificates/${server_node_names[i]}-key.pem -out /tmp/wazuh-certificates/${server_node_names[i]}.csr -config /tmp/wazuh-certificates/${server_node_names[i]}.conf -days 3650 ${debug}"
             eval "openssl x509 -req -in /tmp/wazuh-certificates/${server_node_names[i]}.csr -CA /tmp/wazuh-certificates/root-ca.pem -CAkey /tmp/wazuh-certificates/root-ca.key -CAcreateserial -out /tmp/wazuh-certificates/${server_node_names[i]}.pem -extfile /tmp/wazuh-certificates/${server_node_names[i]}.conf -extensions v3_req -days 3650 ${debug}"
         done
+        cert_setpermisions
```

Comment:
I would change the function call to this place:

https://github.com/wazuh/wazuh-packages/blob/2e1630dd0290742ef43149b5dae2ec90c25d6711/unattended_installer/cert_tool/certMain.sh#L196-L201